Rakefile

```diff
@@ -13,10 +13,9 @@ Rake::TestTask.new(:test) do |t|
 end
 
 desc 'Generate documentation for the ancestry plugin.'
-Rake::RDocTask.new(:rdoc) do |rdoc|
-  rdoc.rdoc_dir = 'doc'
-  rdoc.title    = 'Ancestry'
-  rdoc.options << '--line-numbers' << '--inline-source'
-  rdoc.rdoc_files.include('README.rdoc')
-  rdoc.rdoc_files.include('lib/**/*.rb')
+YARD::Rake::YardocTask.new do |t|
+ t.files   = ['README.rdoc', 'lib/**/*.rb']
```

Comment:
it aligns with the line below it `t.options`

looks like they had a single line indent - will fix